tailwind.config.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
   purge: [
-    "./src/components/*.vue"
+    "./src/components/*/*.vue"
```

Comment:
This turned out to be the big culprit. Since I changed the file structure the tailwind config file wasn't looking for components in subfolders and thus all of the css was not working. Everything is fixed now